modules/developers-guide/pages/work-with-languages.adoc

```diff
@@ -173,17 +173,17 @@ To modify the template source code:
 As you write your program, keep in mind that there are two types of calls—update calls and query calls—and that update functions use asynchronous messaging.
 . Save your changes and close the `+main.rs+` file.
 
-=== Connect to a network and deploy
+=== Connect to an execution environment and deploy
```

Comment:
I would probably just say "Deploy the dapp" -- as connecting is just a precondition for that and connecting to an EE seems weird.